packages/terra-site/src/examples/tag/Index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-tag/docs/README.md';
+import { version } from 'terra-tag/package.json';
+
+// Component Source
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import TagSrc from '!raw-loader!terra-tag/src/Tag';
+
+// Example Files
+import TagDefault from './TagDefault';
+import TagIconAndText from './TagIconAndText';
+import TagHref from './HrefTag';
+
+const TagExamples = () => (
+  <div>
+    <div id="version">Version: {version}</div>
+    <Markdown id="readme" src={ReadMe} />
+    <PropsTable id="props" src={TagSrc} />
+    <h2 id="default">Tag</h2>
+    <TagDefault />
+    <h2 id="icon-and-text">Icon and Text Tag</h2>
```

Comment:
To be honest, I'm ok with it for these examples. Our other example pages have this same structure, and since they all correlate to a subset of a Tag, I think the h2 is appropriate here.